app/src/androidTest/java/org/wikipedia/tests/SearchEngineTests.kt

```diff
@@ -0,0 +1,81 @@
+package org.wikipedia.tests
+
+import androidx.test.ext.junit.rules.ActivityScenarioRule
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import junit.framework.Assert.assertEquals
+import junit.framework.Assert.assertTrue
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.wikipedia.main.MainActivity
+import org.wikipedia.pages.*
+
+@RunWith(AndroidJUnit4::class)
+class SearchEngineTests {
+    val explorePage = ExplorePage()
+    val searchBarPage = SearchBarPage()
+    val searchPage = SearchPage()
+    val navBarPage = NavBarPage()
+    val onboardingPage = OnboardingPage()
+    val articlePage = ArticlePage()
+
+    @Before
+    fun beforeTests() {
+        onboardingPage.tapOnSkipButton()
+    }
+
+    @Rule
+    @JvmField
+    var mActivityTestRule = ActivityScenarioRule(MainActivity::class.java)
+
+
+    @Test
+    fun searchBarTest() {
+        var text = "Donald Trump"
+        explorePage.tapOnSearchBar()
+        searchBarPage.typeTextSearch(text)
+        assertTrue("No view matched '$text'", searchBarPage.isDisplayedElement())
+       }
+
+    @Test
+    fun searchPageTest() {
```

Comment:
Similar to above comment - what does this test do? :)

Reply:
Chnaged the name